src/Stratis.Bitcoin.Features.MemoryPool/Rules/CheckRateLimitMempoolRule.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+
+namespace Stratis.Bitcoin.Features.MemoryPool.Rules
+{
+    /// <summary>
+    /// Validates the rate limit.
+    /// Currently not implemented.
+    /// </summary>
+    public class CheckRateLimitMempoolRule : MempoolRule
+    {
+        public CheckRateLimitMempoolRule(Network network,
+            ITxMempool mempool,
+            MempoolSettings mempoolSettings,
+            ChainIndexer chainIndexer,
+            ILoggerFactory loggerFactory) : base(network, mempool, mempoolSettings, chainIndexer, loggerFactory)
+        {
+        }
+
+        public override void CheckTransaction(MempoolValidationContext context)
+        {
+            // Whether to limit free transactions:
+            // context.State.LimitFree
```

Comment:
Does this have an implementation?

Reply:
No, not even in the existing mempool. I've left the stub so that it doesn't get forgotten about